webtester-core/src/main/java/info/novatec/testit/webtester/mouse/SequenceWithFragment.java

```diff
@@ -0,0 +1,139 @@
+package info.novatec.testit.webtester.mouse;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+
+import info.novatec.testit.webtester.pagefragments.PageFragment;
+
+
+/**
+ * Provides the means of creating a sequence of mouse actions.
+ * This is mainly used for readable fluent API calls.
+ * <p>
+ * <b>Examples:</b>
+ * <pre>
+ * Mouse.sequence().moveTo(image).click();
+ * Mouse.sequence().moveTo(image).moveTo(otherImage).click();
+ * Mouse.sequence().click(image).doubleClick(otherImage);
+ * </pre>
+ *
+ * @see MouseDriver
+ * @see PageFragment
+ * @since 2.0
+ */
+@Getter(AccessLevel.PACKAGE)
+public class SequenceWithFragment {
```

Comment:
@bitterblue theoretically this class could extend Sequence. Which would allow us to remove 4 methods since they are 1:1 copies of the corresponding Sequence implementations. What do you think?